src/CFamily.UnitTests/Rules/DynamicCFamilyRulesConfigTests.cs

```diff
@@ -318,15 +341,20 @@ public void EffectiveParameters_CustomSettingsOverrideDefaults()
 
         private static DynamicCFamilyRulesConfig CreateTestSubject(ICFamilyRulesConfig defaultConfig,
             RulesSettings customSettings,
-            IRulesConfigFixup fixup = null)
+            IRulesConfigFixup fixup = null,
+            IHotspotAnalysisConfiguration hotspotAnalysisConfiguration = null)
         {
             fixup ??= new NoOpRulesConfigFixup();
-            return new DynamicCFamilyRulesConfig(defaultConfig, customSettings, new TestLogger(), fixup);
+            var analysisConfiguration = new Mock<IHotspotAnalysisConfiguration>();
```

Comment:
Always creating an instance of `analysisConfiguration` and then not using it in some cases looks a bit messy and misleading. The normal approach in other tests is to only create an object with default behaviour if one is not supplied.

Reply:
My bad it was meant to be removed before commiting